examples/sdk-examples/src/examples/attributeFilter/index.tsx

```diff
@@ -118,6 +129,12 @@ export const AttributeFilter = (): JSX.Element => (
             reduced via parentFilterOverAttribute property to create parent-child dependency between filters.
         </p>
 
+        <p>
+            <strong>
+                ParentFilter property MUST be a filter defined by uris because of backend limitations.
```

Comment:
no problem with the limitation (for now) however i suggest to reword this; just say something like "current limitation is that the parent filter must be specify attribute elements using their URI"